app/assets/javascripts/lib/models/input_element.coffee

```diff
@@ -6,6 +6,7 @@ class @InputElement extends Backbone.Model
       element: $("#input_element_#{@get('id')}")
     @bind('change:user_value', @markDirty)
     @bind('change:user_value', @logUpdate)
+    @bind('change:user_value', @googleAnalytics)
```

Comment:
To avoid binding this in every `InputElement`, this can be done in the `InputElementList` instead.

```js
class InputElementList extends Backbone.Collection {
  constructor() {
    // ...

    this.on('change:user_value', (inputEl) {
      if (ga && ga.loaded) {
        ga('send', 'event', 'slider-change', inputEl.key);
      }
    });
  }
```